lib/source/encoding.pm

```diff
@@ -0,0 +1,72 @@
+package source::encoding;
+
+use strict;
+use warnings;
+
+our $VERSION = '0.01';
+
+our $ascii_hint_bits = 0x00000010;
+
+sub import {
+    unimport();     # Get rid of any 'use utf8'
+    my (undef, $arg) = @_;
+    if (lc $arg eq 'utf8') {
```

Comment:
Technically speaking, shouldn't case-insensitive comparisons look something like `fc($arg) eq fc('utf8')`?

Reply:
Yes.  Do you want this to be changed?

Reply:
TBH I'd prefer if it were case sensitive. :-)

But I'll leave the decision to you.

Reply:
changed to require all lc